src/contexts/SiteLaunchContext.tsx

```diff
@@ -83,7 +133,8 @@ export const SiteLaunchProvider = ({
       // invalidate react query key
       setTimeout(() => {
         refetchSiteLaunchStatus()
-      }, 120000)
+        //! TODO: change this back to 120000 after local dev
```

Comment:
leaving comment here to remember before merge

Reply:
oh my this was committed by accident 11770b8